utils/oscap_docker_python/oscap_docker_util.py

```diff
@@ -155,7 +155,8 @@ def oscap_chroot(self, chroot_path, target, *oscap_args):
             sys.stderr.write(oscap_stderr.decode("utf-8") + "\n")
 
             # Clean up
-            self._cleanup_by_path(chroot_path)
+            DM = DockerMount("/tmp")
```

Comment:
I think this shouldn't be here, it can be a reason of my problem in my comment below.

Reply:
This code path runs if oscap returns exit code 1.

Reply:
I see, this actually needs to be changed. I will refactor the error handling.